chain/client/src/info.rs

```diff
@@ -144,10 +147,12 @@ impl InfoHelper {
 
         let validator_info_log = validator_info.as_ref().map(|info| {
             format!(
-                " {}{} validator{}",
+                " {}{} validator{} {} block and chunk validators {} chunk only validators",
```

Comment:
Maybe it’s just me but now it feels too verbose.  The status line is annoyingly long as is, so trying to be compact has value IMO.  Perhaps ‘{} validators (of which {} chunk only)’?

Reply:
resolved in [32a96b4](https://github.com/near/nearcore/pull/7788/commits/32a96b4826a9a52525d9b9643fb53ee18a14822b)

Reply:
> Maybe it’s just me but now it feels too verbose.

I do share this concern. Human attention and screen real estate a very scarce resources, any kind of human-oriented output should strive to:

* express everything it needs to express
* clearly and self-explanatory
* with as few symbols as possible